eox_core/edxapp_wrapper/backends/users_m_v1_test.py

```diff
@@ -0,0 +1,104 @@
+"""
+Test backend to get CourseEnrollment Model.
+"""
+
+from unittest.mock import Mock
+
+from django.contrib.auth.models import Permission
+
+USERNAME_MAX_LENGTH = 30
+
+
+def get_edxapp_user(**kwargs):
+    """
+    Return a fake user
+    """
+    return Mock()
+
+
+def create_edxapp_user(*args, **kwargs):
+    """
+    Return a fake user and a list of errors
+    """
+    return Mock(), []
+
+
+def get_user_read_only_serializer():
+    """
+    Return a fake user read only serializer
+    """
+    try:
+        from openedx.core.djangoapps.user_api.accounts.serializers import \
+            UserReadOnlySerializer  # pylint: disable=import-outside-toplevel
+    except ImportError:
+        UserReadOnlySerializer = object
+    return UserReadOnlySerializer
+
+
+def check_edxapp_account_conflicts(email, username):
+    """
+    Get an executes the check_account_exists for tests
+    """
+    try:
+        from openedx.core.djangoapps.user_api.accounts.api import \
+            check_account_exists  # pylint: disable=import-outside-toplevel
+    except ImportError:
+        def check_account_exists(email=None, username=None):
+            """
+            Fake definition for check_account_exists edxapp function
+            """
+            return email and username
+    return check_account_exists(email=email, username=username)
+
+
+def get_course_enrollment():
+    """
+    Get Test CourseEnrollment model.
+
+    We return any django model that already exists so that
+    django-filters is happy and no migrations are created.
+    """
+    return Permission
+
+
+def get_course_team_user(*args, **kwargs):
+    """
+    Return a fake course team user
+    """
+    return Mock()
+
+
+def get_user_signup_source():
+    """
+    Get test UserSignupSource model
+    """
+    try:
+        from common.djangoapps.student.models import UserSignupSource  # pylint: disable=import-outside-toplevel
+    except ImportError:
+        UserSignupSource = object
```

Comment:
```suggestion
        UserReadOnlySerializer = Mock()
    return UserReadOnlySerializer


def check_edxapp_account_conflicts(email, username):
    """
    Get an executes the check_account_exists for tests
    """
    try:
        from openedx.core.djangoapps.user_api.accounts.api import \
            check_account_exists  # pylint: disable=import-outside-toplevel
    except ImportError:
        def check_account_exists(email=None, username=None):
            """
            Fake definition for check_account_exists edxapp function
            """
            return email and username
    return check_account_exists(email=email, username=username)


def get_course_enrollment():
    """
    Get Test CourseEnrollment model.

    We return any django model that already exists so that
    django-filters is happy and no migrations are created.
    """
    return Permission


def get_course_team_user(*args, **kwargs):
    """
    Return a fake course team user
    """
    return Mock()


def get_user_signup_source():
    """
    Get test UserSignupSource model
    """
    try:
        from common.djangoapps.student.models import UserSignupSource  # pylint: disable=import-outside-toplevel
    except ImportError:
        UserSignupSource = Mock()
```
Is there any reason not to change those objects for mocks? Please let me know.

Reply:
These change broke the tests because we need a specific mock, we can do it in a future commit.